app/Http/Controllers/Tabbar/Navigation/Home/HomeController.php

```diff
@@ -3,29 +3,21 @@
 namespace App\Http\Controllers\Tabbar\Navigation\Home;
 
 use App\Http\Controllers\Controller;
-use App\Settings\PathInfo;
 
-class HomeController extends Controller
-{
-    /*
-    |--------------------------------------------------------------------------
-    | HomeController
-    |--------------------------------------------------------------------------
-    |
-    | Homeコントローラ
-    |
-    */
+/**
+ * HomeController
+ * 
+ * Homeタブ内で表示するコンテンツを扱う
+ */
+class HomeController extends Controller {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition